static/src/stylesheets/module/email-signup/_form.scss

```diff
@@ -213,29 +214,29 @@
 .email-sub__thrasher-embed-row {
     display: flex;
     justify-content: flex-start;
-    align-items: center;
+    height: 36px;
+    align-items: stretch;
 }
 
 .email-sub__thrasher-embed-input {
     @include fs-textSans(3, true);
-    margin-right: 4px;
-    margin-bottom: 4px;
+    margin-right: 8px;
     flex: 1;
-    max-width: 18.5rem;
-    min-width: 7rem;
+    max-width: 296px;
+    min-width: 112px;
```

Comment:
If using `flex`, you can probably avoid specifying these values if you set `flex-basis`. Why do we need a maximum width?

Reply:
The form will be in a iframe in a responsive container as part of an atom so you don't really get the full context from this repo. 
The iframe's width changes at different breakpoints. The design calls for the input field to take up all available horizontal space (allowing for the button next to it in the row ) in the iframe at smaller widths (hence flex:1) but be limited to 296px when the iframe is wide enough to fit a wider input field.